src/storage/documentService/documentService.js

```diff
@@ -47,17 +50,30 @@ const validateTtl = ttl => {
 
 const uploadDocument = async (
   document,
+  docId,
```

Comment:
why not set `docId = uuid()` as a default param then you won't need to reassign it to handle eslint

Reply:
currently I am checking if docId is there then only make the call to getDocument to find the existing document. If i set it as default I will have to make call every time.

Reply:
i found a better solution using conditional expressions
https://www.dynamodbguide.com/expression-basics/
https://stackoverflow.com/questions/53915405/how-to-check-and-then-update-aws-dynamodb-item-in-node-js